src/Component.php

```diff
@@ -236,6 +236,9 @@ private function getExtractor(): Extractor
     private function getGoogleRestApi(): RestApi
     {
         $tokenData = json_decode($this->getConfig()->getOAuthApiData(), true);
+        if (!isset($tokenData['access_token'], $tokenData['access_token'])) {
```

Comment:
zdá se mi to nebo tam testuješ dva stejný klíče?

Reply:
🤦 fixed, díky!